ambari-server/src/main/java/org/apache/ambari/server/controller/internal/ComponentResourceProvider.java

```
@@ -549,11 +544,9 @@ private ServiceComponentRequest getRequest(Map<String, Object> properties) {
       ServiceComponentResponse serviceComponentResponse = sc.convertToResponse();
       StackId stackId = sc.getStackId();
 
-      ServiceGroup sg = null;
       try {
-        sg = cluster.getServiceGroup(sc.getServiceGroupId());
+        cluster.getServiceGroup(sc.getServiceGroupId());
       } catch (ServiceGroupNotFoundException e) {
-        sg = null;
       }
```

Looks like a no-op, now that both return value and exception are ignored.